HackerNoon/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+"author": {
+"name": "Bünyamin Öztürk",
+"id": "487515609815580672"
+},
+"service": "HackerNoon",
+"description": {
+"en": "HackerNoon is an Independent Tech Media Site",
+"tr": "HackerNoon, Bağımsız bir Teknik Medya Sitesidir"
+},
+"url": [
+"www.hackernoon.com",
+"hackernoon.com"
+],
+"version": "1.0.1",
+"logo": "https://hackernoon.com/Devimg/hn-logo.png",
+"thumbnail": "https://thetokenist.io/wp-content/uploads/2019/02/hackernoon-logo.png",
```

Comment:
please upload your images to a site like imgur, hackermoon can change their logo on their any second to any size or delete it completly and break something.

Reply:
Done 👌